### PR TITLE
Adds Cricket Borgs to appearance selection for Cyborgs.

### PR DESCRIPTION
**Pull request by DesolateG**
Adds new appearence selections for Security, Medical, Mining, Engineering, Janitor, and Service silicons.

Introducing: The Cricket.

dreamseeker_2018-03-23_02-25-17

This PR also allows for use of other Panel sprites for public cyborg sprites based in the robots.dmi and robots.dm files, while having those not listed in this format to retain the default open Panel sprites.

dreamseeker_2018-03-23_02-32-20

dreamseeker_2018-03-23_02-35-12

Thanks to Jet on discord for donating these sprites for use, and a thanks to @shazbot194  for assisting with the coding.

:cl: Desolate
add: New Cricket Borg module selection.
/:cl:

### CODE REVIEW

**Comment by Spacemanspark** [comment]
I'm not really a fan of the sprites personally, but honestly if enough people want it then why not

**Comment by RyanSmake** [comment]
> add: Now capable of using additional panels other than the default open panel sprites.

What regular players don't see don't really need a CL.
Otherwise +1

**Comment by Spartan6** [comment]
PLEASE PUT DOWN YOUR WEAPON, YOU HAVE 15 SECONDS TO COMPLY.

**Comment by FalseIncarnate** [comment]
Definitely reminiscent of the EDs and also has a bit of a mecha look to them, but skinny enough where they shouldn't be confused with a mech too commonly. My only suggestion (and this might not really be that big a deal) is to shorten them slightly and give their little gun barrel a bit more variety between variants (janitor already has this, but consider making the mining version have more of a drill, or the medical one more of a syringe, or engineering having little t-rex arms for example)

Would I be correct that the "eye" changes color when emagged?

**Comment by SilvernKell** [comment]
I agree with the different mechanisms on the front, however I find it kind of hard to fit anything on there that is readable as a drill etc, since it's such a small amount of pixels! Especially on the front. However as they are public use if anyone can improve upon them that's totally legit too.
Eyes changing when emagged? As far as I know, emagged borgs eyes stay the same : also the goal of being emagged is generally to blend in and having red eyes or something would completely ruin any chances of that one

**Comment by FalseIncarnate** [comment]
Fair enough, was just a thought on potential improvements to help make them more distinct, but totally understand the constraints of the sprite size makes that difficult.

I'm personally going to be calling that janiborg yog-borgoth though.

**Comment by SilvernKell** [comment]
Hahaha xD It should be called that , yeah.

**Comment by Fox-McCloud** [comment]
These look pretty ugly, in my opinion.

**Comment by taukausanake** [comment]
I think they look nice. ED wannabes but pretty detailed as well. Their shoulders look like gems though

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -1289,6 +1300,8 @@ var/list/robot_verbs_default = list(
 			module.module_type = "Brobot"
 			update_module_icon()
 		lockcharge = null
+		var/list/names = splittext(icontype, "-")
+		custom_panel = trim(names[1])
```
Comment (Crazylemon64) [inline code/modules/mob/living/silicon/robot/robot.dm]: hold on, how does this respond to skins that lack a custom panel?

**Pull request merged**